docs/python/inference/api_summary.rst

```diff
@@ -133,33 +165,46 @@ Users can bind *OrtValue* (s) directly.
 	io_binding.bind_ortvalue_output('output', Y_ortvalue)
 	session.run_with_iobinding(io_binding)
 
-Device
-======
-
-The package is compiled for a specific device, GPU or CPU.
-The CPU implementation includes optimizations
-such as MKL (Math Kernel Libary). The following function
-indicates the chosen option:
 
-.. autofunction:: onnxruntime.get_device
+You can also bind inputs and outputs directly to a PyTorch tensor.
 
-Examples and datasets
-=====================
-
-The package contains a few models stored in ONNX format
-used in the documentation. These don't need to be downloaded
-as they are installed with the package.
-
-.. autofunction:: onnxruntime.datasets.get_example
+.. code-block:: python
 
-Load and run a model
-====================
+    # X is a PyTorch tensor on device
+    session = onnxruntime.InferenceSession('model.onnx')
```

Comment:
For all the examples under "Data on device" I would've expected the InferenceSession to be constructed using the 'cuda' argument. The current code will not place any operators on CUDA, so the mention of a non-CPU device becomes moot. But I guess this is not introduced in this PR. cc @hariharans29

Reply:
>  InferenceSession to be constructed using the 'cuda' argument.

What is the 'cuda' argument to InferenceSession? Can you please summarize here? @pranavsharma

Reply:
onnxruntime.InferenceSession('model.onnx', providers=['CUDAExecutionProvider'])

Reply:
I meant specifying the cuda provider in the argument. This is exactly what George is saying as well below.

Reply:
@pranavsharma - I think that is because these docs were crafted initially when the provider list was constructed based on build flavor and the user didn't necessarily have to provide the provider list.

Reply:
Added providers to each of the arguments.